pkg/client/typed/dynamic/client.go

```diff
@@ -49,10 +49,6 @@ func NewClient(conf *restclient.Config) (*Client, error) {
 
 	conf.Codec = dynamicCodec{}
 
-	if conf.APIPath == "" {
```

Comment:
Made some changes, please take a look now. I hope it is clear how scaler is using dynamic client.